src/common.rs

```diff
@@ -318,6 +319,35 @@ impl SectionId {
         }
     }
 
+    /// Returns the XCOFF section name for this kind.
+    #[cfg(target_os = "aix")]
```

Comment:
This library supports parsing non-native binaries, so we can't change the meaning of a method like this (changing it from ELF name to XCOFF name based on target). Instead, we need to keep the `name` method unchanged, and add a new method such as `xcoff_name` that returns the XCOFF name.